packaging/windows/clisdk/generatemsi.ps1

```diff
@@ -26,7 +27,19 @@ function RunHeat
 
     Write-Output Running heat..
 
-    .\heat.exe dir `"$inputDir`" -template fragment -sreg -gg -var var.DotnetSrc -cg InstallFiles -srd -dr DOTNETHOME -out $InstallFileswsx | Out-Host
+    # -t $StableFileIdForApphostTransform to avoid sign check whitelist apphost.exe name changes very build. Sign check uses File Id in MSI as whitelist name.
```

Comment:
Change "whitelist" to "baseline", and "very" to "every".

Reply:
Fixed